test/uk/gov/hmrc/minorentityidentificationfrontend/services/ValidateTrustKnownFactsServiceSpec.scala

```diff
@@ -42,7 +42,7 @@ class ValidateTrustKnownFactsServiceSpec extends AnyWordSpec with Matchers with
         mockStorageService.storeTrustsKnownFacts(testJourneyId, testTrustKnownFactsResponse) returns Future.successful(SuccessfullyStored)
         mockStorageService.storeIdentifiersMatch(testJourneyId, SuccessfulMatch) returns Future.successful(SuccessfullyStored)
 
-        val result = await(TestJourneyService.validateTrustKnownFacts(testJourneyId, testSautr, Some(testSaPostcode)))
+        val result = await(TestJourneyService.validateTrustKnownFacts(testJourneyId, Some(testSautr), Some(testSaPostcode), Some(testCharityHMRCReferenceNumber)))
```

Comment:
CHRN should never exist if the SAUTR exists

Reply:
okay